src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -206,6 +207,7 @@
     <HelixPreCommand Include="set __CollectDumps=1" />
     <HelixPreCommand Include="set __CrashDumpFolder=%HELIX_DUMP_FOLDER%" />
     <HelixPreCommand Include="type %__TestEnv%" />
+    <HelixPreCommand Include="set MONO_ENV_OPTIONS='--interpreter'" Condition=" '$(RuntimeMode)' == 'interpreter' " />
```

Comment:
I think the TestRunNamePrefix should also contain `interpreter` in the name to make it easier to identify if there is a failure in the azure devops test explorer.